### ISSUE TITLE
hard coded moduleid

### ISSUE THREAD

**Issue by danmarsden** [pwsdb/moodle-block_remaining_lesson_timer#6]
https://github.com/pwsdb/moodle-block_remaining_lesson_timer/blob/master/block_remaining_lesson_timer.php#L87

unfortunately you can't guaruntee the moduleid will be 11 in all sites - you should probably join with the modules table to get the module id instead.

**Comment by pwsdb** [comment]
Right.  I see that as of moodle 3.0 (?) Perth has chosen to change the lesson record number from 11 to 13 for new installs.   I do want to add the line-link to check for the correct record #.  What is the best way to avoid another 2 month wait for re-approval?  Thanks.

**Comment by danmarsden** [comment]
Quite often a site will have a number of other 3rd party modules on fresh install too, - the Id field is basically just the order of module install.

Re-reviews with minor changes shouldn't sit in the review queue too long.

**Comment by alivin70** [comment]
I pushed a commit and a pull request to fix this issue
